.github/workflows/build_and_test_workflow.yml

```diff
@@ -95,18 +96,21 @@ jobs:
         run: yarn osd bootstrap || yarn osd bootstrap
 
       - name: Run linter
+        if: matrix.group == 2 && matrix.os == 'ubuntu-latest'
```

Comment:
Should we add a comment that a lot of this conditional logic is premised on the assumptions that:

1. linux tests run faster than windows tests
2. ci group 2 is generally faster than ci group 1, so that's where we should add additional checks

(Interestingly, on my local, ci group 1 was faster - 81 vs 86 seconds - but what matters is the general speed as ran by the workflow)

Reply:
That meant the balancing was successful where 1 took roughly the same time as 2 + linter + ...

Reply:
No, this was just running them locally, so it was just the jest components.